package.json

```diff
@@ -40,6 +40,7 @@
   "dependencies": {
     "@11ty/eleventy": "^0.12.1",
     "@11ty/eleventy-navigation": "^0.3.2",
+    "@11ty/eleventy-plugin-rss": "^1.1.2",
```

Comment:
Might not be needed, if this is just for string formatting? See above.

Reply:
I was going to add some Atom/RSS stuff at a later date, but happy to remove it.

Reply:
Remove until we add it?